src/metax_api/services/catalog_record_service.py

```diff
@@ -52,61 +52,90 @@ def get_queryset_search_params(cls, request):
         Get and validate parameters from request.query_params that will be used for filtering
         in view.get_queryset()
         """
+        queryset_search_params = {}
 
         if not request.query_params:
-            return {}
+            queryset = cls.filter_by_state(request, {})
+            return queryset
```

Comment:
It is best to return directly ``return cls.filter_by_state(request, {})`` instead of using the extra variable in between. (the returned value is not a ``queryset`` btw, but ``queryset_search_params``.)

Additionally, instead of passing ``{}`` as an argument to the ``filter_by_state()`` method, pass ``queryset_search_params``. That way, in case ``queryset_search_params`` ever changes before this block, this method call does not have to be changed.